src/obfuscation.ts

```diff
@@ -12,3 +14,22 @@ export function encodeBase64(input: string) {
 export function decodeBase64(input: string) {
   return base64.atobPolyfill(input);
 }
+
+export function obfuscatePrecomputedFlags(
+  precomputedFlags: Record<string, PrecomputedFlag>,
+): Record<string, PrecomputedFlag> {
+  const response: Record<string, PrecomputedFlag> = {};
+  Object.keys(precomputedFlags).map((flagKey) => {
+    const assignment = precomputedFlags[flagKey];
+
+    response[getMD5Hash(flagKey)] = {
+      variationType: assignment.variationType,
+      extraLogging: assignment.extraLogging,
+      doLog: assignment.doLog,
+      allocationKey: getMD5Hash(assignment.allocationKey),
+      variationKey: getMD5Hash(assignment.variationKey),
```

Comment:
These values will end up in the customers' datawarehouse and need to be coherent (original); since md5 is not reversible our only choice is to use base64. Obvious this leaves is reversal publicly but our goal is to defeat just simple snooping.

Reply:
we have `obfuscation.ts` where some stuff already exists and I've now added more there.